HPXMLtoOpenStudio/resources/location.rb

```diff
@@ -81,6 +81,37 @@ def self.get_climate_zone_iecc(wmo)
     return
   end
 
+  def self.get_supplement_table_short_grass
+    supplement_table_short_grass_csv = File.join(File.dirname(__FILE__), 'data', 'Supplement_Table-Short-Grass-US.csv')
+    if not File.exist?(supplement_table_short_grass_csv)
+      fail 'Could not find Supplement_Table-Short-Grass-US.csv'
+    end
+
+    return supplement_table_short_grass_csv
+  end
+
+  def self.get_xing_amplitudes(ts_amp_1, ts_amp_2, pl_1, pl_2, weather_header)
+    supplement_table_short_grass = get_supplement_table_short_grass
+
+    require 'csv'
+    require 'matrix'
+
+    v1 = Vector[weather_header.Latitude, weather_header.Longitude]
+    dist = 1 / Constants.small
+    xing_amplitudes = nil
+    CSV.foreach(supplement_table_short_grass) do |row|
+      v2 = Vector[row[3].to_f, row[4].to_f]
+      new_dist = (v1 - v2).magnitude
+      if new_dist < dist
+        xing_amplitudes = row[6..9].map(&:to_f)
+        dist = new_dist
+      end
+    end
+    return [ts_amp_1, ts_amp_2, pl_1, pl_2] if dist > 1 # return default values; not close enough
```

Comment:
Return the default values when lat/lon does not match anything close enough in the table? What should the threshold be? For example, I think basically any non-US location would get default values.

Reply:
Alaska is going to be tough. It spans more than 140W - 165W and 55N - 70N. A guess would be 3? See the map of TMY3 locations (red) in the figure attached.
![Screen Shot 2023-10-26 at 12 38 57 PM](https://github.com/NREL/OpenStudio-HPXML/assets/37630588/334b6cf2-6a0c-4441-a549-267fe0c4d279)